content/en/docs/concepts/cluster-administration/logging.md

```diff
@@ -220,6 +221,10 @@ of this approach is a small container running logrotate periodically.
 However, it's recommended to use `stdout` and `stderr` directly and leave rotation
 and retention policies to the kubelet.
 
+{{< note >}}
+This works because the side car will be restarted until the file exists. For more advance usage have a look to `ls app.log && tail -F app.log` is more powerfull for legacy application doing daily rolling.
```

Comment:
nit:
```suggestion
This works because the sidecar will be restarted until the file exists. For more advanced usage have a look to `ls app.log && tail -F app.log` is more powerful for legacy application doing daily rolling.
```

@NargiT I don't think I'd put this in a `{{< note >}}` block. It's an aside from the text but to me it's different from what `{{< note >}}` is for.

What's “daily rolling”? Does that mean the same as: “for an application that writes to a log and rotates that log daily.”?

Reply:
Yes, daily rolling means that. 
I tried to provide an example that works in case you have "rolling logs" logic provided by the application. Most of the java applications implements rolling files out of the box and in our company we are using this technique to avoid updating all our applications.

Should I remove  `{{< note >}}` ?